es6-async-await/await.js

```diff
@@ -0,0 +1,56 @@
+import fetch from './fetch.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = () => `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+async function fetchOnce() {
+  try {
+    const response = await fetch('foo/bar.html');
+    const msg = await response.toString();
+    console.log(elapsed(), 'fetchOnce:', msg);
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+async function fetchSeveral() {
+  try {
+    const response = await fetch('foo1/bar.html');
+    const msg = await response.toString();
+    console.log(elapsed(), 'fetchSeveral1:', msg);
+
+    const response2 = await fetch('foo1/bar.html');
+    const msg2 = await response2.toString();
+    console.log(elapsed(), 'fetchSeveral2:', msg2);
+
+    const response3 = await fetch('foo1/bar.html');
+    const msg3 = await response3.toString();
+    console.log(elapsed(), 'fetchSeveral3:', msg3);
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+async function fetchChained() {
+  try {
+    const response = await fetch('foo-chain/bar.html');
+    const msg = await response.toString();
+    console.log(elapsed(), 'fetchChained1:', msg);
+
+    const response2 = await fetch('foo-chain/bar.html');
```

Comment:
This also isn't quite how the original code is written. Note that in that code when you make this 2nd fetch call, you are passing it the response from the first fetch.

Reply:
This one still isn't fixed. The original code calls `fetch(msg)` here and `fetch(msg2)` below.